include/ddprof_module.hpp

```diff
@@ -29,6 +31,12 @@ struct DDProfMod {
 
   explicit DDProfMod(Status status) : DDProfMod() { _status = status; }
 
+  void set_build_id(std::span<const unsigned char> x) {
```

Comment:
Why not accept directly a `BuildIdStr` and do the conversion when retrieving it from dwfl ?

Reply:
I don't like having it as a string and still want to leave the path open to have this as a number